clockwork_web/templates/jobs_search.html

```diff
@@ -182,10 +182,10 @@ <h1>JOBS</h1>
                             {% endif %}
 
                             <!-- Job array -->
-                            <!-- TODO For now, job array is just job ID. To update when real job arrays will be available -->
+                            <!-- NB: If job array is unset, we use value zero (0). -->
                             {% if (web_settings | check_web_settings_column_display(page_name, "job_array")) %}
                                 <td>
-                                    <a href="{{ modify_query(job_array=D_job['slurm']['job_id']) }}" title="Filter by job array {{D_job['slurm']['job_id']}}">
+                                    <a href="{{ modify_query(job_array=(D_job['slurm']['array_job_id'] or '0')) }}" title="Filter by job array {{D_job['slurm']['array_job_id'] or '0'}}">
```

Comment:
I would add a `page_num=1` to the request `modify_query(job_array=(D_job['slurm']['array_job_id'] or '0'), page_num=1)` in order to get this list of jobs from the beginning.

I think we could simply not display the icon if the job does not belong to a job array, by adding a condition as this:
```
<td>
{% if D_job['slurm']['array_job_id'] == "0" or D_job['slurm']['array_job_id'] is none %}
  <a href="{{ modify_query(job_array=(D_job['slurm']['array_job_id'] or '0')) }}" title="Filter by job array {{D_job['slurm']['array_job_id'] or '0'}}">
{% endif %}
</td>
```

Reply:
Fait!